### DISCUSSION TITLE
My updated code push to master branch and then deleted the master branch

### DISCUSSION THREAD

**Discussion by idrees-official** [community/community#120213] [Code Security]
### Select Topic Area

General

### Body

Dear Sir/Mada,
my flutter code was not pushing in main branch then github suggest me to push into origin master, then i push it into the master branch and due to mistake i deleted master branch of my project, and locally due to space issue i have deleted my local project too, 
now i want to recover my master branch which is deleted how can i recover that branch because it was my hard work of about 6 months.

**Comment by SoroushBeigi** [comment]
hello, the easiest solution is to Open the repository in GitHub where the branch was deleted.

1. Click on the "Code" tab to view the repository's code.

2. In the dropdown menu next to the branch selector, click on the branch selector and select the branch that was deleted.

GitHub will display a message indicating that the branch was deleted and provide an option to restore it.

3. Click on the "Restore branch" button.

4. GitHub will recreate the deleted branch with all its commit history.
